homeassistant/components/sfr_box/sensor.py

```diff
@@ -148,25 +148,23 @@ async def async_setup_entry(
     hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
 ) -> None:
     """Set up the sensors."""
-    data = hass.data[DOMAIN][entry.entry_id]
-    box: SFRBox = data["box"]
-    system_info = await box.system_get_info()
+    data: DomainData = hass.data[DOMAIN][entry.entry_id]
 
     entities = [
-        SFRBoxSensor(data["dsl_coordinator"], description, system_info)
+        SFRBoxSensor(data.dsl, description, data.system.data)
```

Comment:
Does this mean that the system data coordinator won't update after the first update, as there are no listeners (entities) that uses that coordinator, currently?

Reply:
Yes that's correct.
For now, we only use the system data coordinator to access the static information (device_info and unique_id)
In a follow-up PR, I will add some extra sensors from the system coordinator.